README.md

```diff
@@ -58,11 +58,17 @@ without a port name or protocol prefix.
 #### `LDAPAuthenticator.bind_dn_template` ####
 
 Template used to generate the full dn for a user from the human readable
-username. This must be set to a non-empty list of templates the
-users belong to. For example, if some of the users in your LDAP database have DN
-of the form `uid=Yuvipanda,ou=people,dc=wikimedia,dc=org` and some other users
-have DN like `uid=Mike,ou=developers,dc=wikimedia,dc=org` where Yuvipanda and
-Mike are the usernames, you would set this config item to be:
+username.
+
+If `bind_dn_template` isn't explicitly configured and `lookup_dn = True`, a
```

Comment:
Maybe this could be worded more strongly, because `lookup_dn` only takes effect if `bind_dn_template` is falsey

Reply:
Oh, i didnt know this!

Reply:
After inspecting the code again, I conclude that lookup_dn will impact the username if lookup_dn is true, no matter if bind_dn_template is truthy or falsy.

I restructured the code slightly by grouping truthy/falsy lookup_dn become easier to overview.